packages/app-project/src/components/AuthModal/shared/components/withFormik/withFormik.spec.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import { expect } from 'chai'
+import { Box, Button, FormField, TextInput } from 'grommet'
+import { Formik } from 'formik'
+import sinon from 'sinon'
+import withFormik from './withFormik'
+
+const MockForm = ({ handleBlur, handleChange, handleSubmit, values }) => (
+  <Box as='form' onSubmit={handleSubmit}>
+    <FormField htmlFor='testInput' label='Enter text' >
+      <TextInput
+        id='testInput'
+        name='testInput'
+        onBlur={handleBlur}
+        onChange={handleChange}
+        type='text'
+        value={values.testInput}
+      />
+    </FormField>
+    <Button type='submit' label='Submit' />
+  </Box>
+)
+
+const WrappedMockForm = withFormik(MockForm)
+
+const initialValues = { testInput: '' }
+const onSubmit = sinon.spy()
+
+describe('Higher Order Component > withFormik', function () {
+  it('renders without crashing', function () {
+    const wrapper = shallow(<WrappedMockForm />)
+    expect(wrapper).to.be.ok()
+  })
+
+  it('should pass Formik required props', function () {
+    const wrapper = shallow(
+      <WrappedMockForm
+        initialValues={initialValues}
+        onSubmit={onSubmit}
+      />
+    )
+
+    const formikComponent = wrapper.find(Formik)
+
+    expect(formikComponent.props().initialValues).to.equal(initialValues)
+    expect(formikComponent.props().onSubmit).to.equal(onSubmit)
+  })
+
+  it('should pass Formik optional props', function () {
+    const validate = sinon.spy()
+    const wrapper = shallow(
+      <WrappedMockForm
+        initialValues={initialValues}
+        onSubmit={onSubmit}
+        validate={validate}
+      />
+    )
+    const formikComponent = wrapper.find(Formik)
+
+    expect(formikComponent.props().validate).to.equal(validate)
+  })
+
+  it('should call onChange prop', function () {
+    const onChangeStub = sinon.stub()
+    const handleChangeStub = sinon.stub()
+    const eventMock = {}
+    const formikPropsMock = {}
```

Comment:
I think this should be
```js
const formikPropsMock = wrapper.dive().find('MockForm').props()
```
so that you're testing the props that Formik generates for the wrapped form.